sdk/core/azure-core/inc/azure/response.hpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Don't move files on disks (even if we decide we want to), makes code reviews harder. **Let's do a first pass of just namespace updates.**